src/metabase/premium_features/token_check.clj

```diff
@@ -55,10 +59,6 @@
              (str/replace  #"/$" "")))
    "https://token-check.metabase.com"))
 
-(def store-url
```

Comment:
Is this fallback url not used anymore? Is the grace period a replacement for a fallback?